pkg/apis/management.cattle.io/v3/schema_types.go

```diff
@@ -5,75 +5,295 @@ import (
 )
 
 // +genclient
-// +kubebuilder:skipversion
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:resource:scope=Cluster
 
+// DynamicSchema is the definition of a schema. Dynamic schemas are
+// created and used internally by Rancher to generate CRDs and API
+// endpoints, and should not be created manually.
+//
+// DynamicSchema functionality for generating Rancher dynamic API
+// endpoints is limited. These endpoints are not backed by any stores,
+// and most related fields will not have any useful effect.
+// ---
+// DynamicSchema is used by Rancher for generating dynamic
+// provisioning and CAPI CRDs for Rancher node drivers: the machine
+// pool configuration CRD, the InfrastructureMachineTemplate CRD, and
+// the InfrastructureMachine CRD. Dynamic schemas are generated by
+// Rancher from node and cluster drivers.
+//
+// DynamicSchema is also used by Rancher to generate meta-schemas that
```

Comment:
Adding `DynamicSchema` as the first word of each paragraph reads a bit odd to me, does this format make more sense when viewed via `kubectl`? Otherwise I think this is written well